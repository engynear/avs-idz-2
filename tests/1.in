test foo test bar test kekes lol 123 idontknowwhattowritehere ifif else telse