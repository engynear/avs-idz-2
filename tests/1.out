if: 2, do: 1, count: 0, for: 0, else: 2